File: TUF/Numbers/CheckEvenOrOddNumber.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function checkEvenOrOdd(num) {
    return num % 2 == 0 ? "Even" : "Odd";
}
var num = 6;
console.log("".concat(num, " is a : ").concat(checkEvenOrOdd(num), " number."));

File: TUF/Numbers/CheckEvenOrOddNumber.java
import java.util.Scanner;

public class CheckEvenOrOddNumber {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number to check: ");
        int number = input.nextInt();
        input.close();
        System.out.println("The number is: " + checkEvenOrOdd(number));
    }

    private static String checkEvenOrOdd(int num) {
        if (num % 2 == 0) {
            return "Even";
        } else {
            return "Odd";
        }
    }
}